src/test/java/org/cactoos/list/ImmutableTest.java

```diff
@@ -560,4 +560,30 @@ public void testToString() {
             new IsEqual<>(new ListOf<>("a", "b", "c").toString())
         ).affirm();
     }
+
+    @Test
+    public void subListReturnsListIteratorWithSupportedSet() {
+        new Assertion<>(
+            "subList.listIterator().set() must throw exception",
+            () -> {
+                final ListIterator<String> iterator = new Immutable<>(
+                    new ListOf<String>("one", "two", "three")
+                )
+                    .subList(0, 2)
+                    .listIterator(0);
```

Comment:
@andreoss please, fix ident here.

Reply:
@fabriciofx I believe the ident is fine, at least qulice does not allow it to be shifted either left or right here.